src/index.html

```diff
@@ -17,11 +17,395 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The stylesheet link references a SCSS file (`styles/style.scss`). Ensure that this SCSS file is compiled to a CSS file and link the resulting CSS file instead, as browsers do not directly interpret SCSS.